python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -823,19 +826,30 @@ async def arun(self, checkpoint: bool = False) -> None:
                 )
                 if checkpoint:
                     await self.checkpoint(
-                        f"[{i + 1}/{self.max_iter}]: CWFS applying coma and focus correction."
+                        f"[{self.iterations_executed + 1}/{self.max_iter}]: CWFS applying coma and focus correction."
                     )
                 self.offset_total_coma_x += coma_x
                 self.offset_total_coma_y += coma_y
                 await self.atcs.offset_aos_lut(z=focus_offset, x=coma_x, y=coma_y)
 
+        # If we reach this point, it means we did not converge.
         self.log.warning(
-            f"Reached maximum iteration ({self.max_iter}) without convergence.\n"
+            f"Reached maximum iteration ({self.max_iter}) without convergence."
+            f"Returning hexapod to initial position. \n"
+        )
+
+        self.rem.ataos.evt_detailedState.flush()
+
+        # Return hexapod to original position
+        await self.atcs.offset_aos_lut(
```

Comment:
I wonder if this is actually what we want to do. In some cases it doesn't converge but it is more aligned than before, in some other cases it might be a divergence of some type. I think most times it is the first case and we are good to keep the offsets. Maybe @edennihy could comment.

Reply:
Good point. If it reaches this point in the loop, it means the new values are bigger than the threshold (coma and focus). That is why I added it here, but maybe we need to add one more condition here to check the real need to return it to its initial position. Or just remove it? Cc @edennihy

Reply:
I think the safest case for now is to return to original position. We should be able to control the frequency of the first case by adjusting the threshold values. Recently, after tightening the focus value threshold we have been seeing more cases where the focus is improving, but the coma is diverging.

Reply:
I think the additional logic we would need to capture that should be a separate ticket after we finish our experiments with the threshold values.

Reply:
@dsanmartim, @tribeiro and I talked through this again at lunch today and decided that the original code behavior is best. If after reaching the iteration limit we still do not converge, we should leave the hexapod in it's current position after the corrections from the final iteration have been applied. 

To put it another way, we do **not** want to return to the original position in the case where we do not converge. Sorry for the confusion!